superbench/analyzer/data_diagnosis.py

```diff
@@ -120,25 +149,14 @@ def _get_criteria(self, rule_file, baseline_file):
             for rule in benchmark_rules:
                 benchmark_rules[rule] = self._check_rules(benchmark_rules[rule], rule)
                 self._sb_rules[rule] = {}
+                self._sb_rules[rule]['name'] = rule
                 self._sb_rules[rule]['function'] = benchmark_rules[rule]['function']
+                self._sb_rules[rule][
+                    'store'] = True if 'store' in benchmark_rules[rule] and benchmark_rules[rule]['store'] else False
```

Comment:
`self._sb_rules[rule]['store'] =` in one line

Reply:
Why do we need `and benchmark_rules[rule]['store']`

Reply:
it's due to auto-format and I mean and benchmark_rules[rule]['store'] is True